src/controllers/pagoPAProxyController.ts

```diff
@@ -32,7 +33,10 @@ export default class PagoPAProxyController {
     | IResponseSuccessJson<PaymentRequestsGetResponse>
   > =>
     withValidatedOrInternalError(
-      t.string.decode(req.params.rptId),
+      GetPaymentInfoProxyRequest.decode({
```

Comment:
you don't need the extra definition here, just decode the parameter:

```suggestion
      withValidatedOrInternalError(t.string.decode(req.params.rptId), rptId => {
        const isTest = String(req.query.test).toLowerCase() === "true";
        return this.pagoPAProxyService.getPaymentInfo(rptId, isTest)
      })  
```

Reply:
changed in #b235f21